<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphViewClassClass</class>
 <widget class="QMainWindow" name="GraphViewClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtApplication1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
				background-color: rgb(243, 243, 243);
				background-color: rgb(255, 255, 255);
				background-color:rgb(34, 34, 34);


			</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1536</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						background-color: rgb(53,50,84);
						background-color: rgba(75, 0, 0, 200);
						background-color: rgb(34,34,34);
					</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-weight: 400;
							font: 20pt &quot;Bahnschrift Condensed&quot;;
						</string>
     </property>
     <property name="text">
      <string>Road-Map</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/img/img/pin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>89</y>
      <width>1531</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>901</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,150);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addCityButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletecitybutton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="city_name1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter City Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addEdgeButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Edge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteedgebutton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete Edge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteall">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete All Graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undobutton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>490</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="redobutton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>490</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Submit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="city_name2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter City Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="road_cost">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter City Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toalgo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>540</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>to algo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
								color: rgb(57, 196, 173);
							</string>
      </property>
      <property name="text">
       <string>Create your graph</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/img/distance.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>921</width>
        <height>351</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>919</width>
         <height>349</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bfs">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>135</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dfs">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prims">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>130</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Prims</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dijkistra">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>135</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dijkistra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kruksal">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>130</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kruksal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tograph">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>620</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
								color: rgb(57, 196, 173);
							</string>
      </property>
      <property name="text">
       <string>Back to graph</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>1021</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
								font-weight: 400;
								font: 40pt &quot;Bahnschrift Condensed&quot;;
								color: rgb(57, 196, 173);


							</string>
      </property>
      <property name="text">
       <string>Apply Algorithms On Your Graph</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/img/algorithm (2).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>1536</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(75, 75, 75);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
